tests/test_config.py

```
@@ -11,0 +1,302 @@
+        yaml_data['email']['sender'] = 's@example.com'
+        yaml_data['email']['smtp_server'] = 'server'
+        yaml_data['email']['smtp_username'] = 'user'
+        yaml_data['paths'] = {'download_dir': '/tmp'}
+        
+        mock_yaml_load.return_value = yaml_data
+        
+        # Re-enable INFO logging for this specific test to capture summary
+        logging.disable(logging.NOTSET) 
+        # Create a new config instance that will use the patched logger
+        test_config = Config()
+        with patch.object(config.logger, 'log', mock_log_method): # Patch logger used by config instance
```

Ruff (0.11.9)

Undefined name `config`

(F821)